projects/js-packages/publicize-components/src/hooks/use-dismiss-notice/index.js

```diff
@@ -2,6 +2,13 @@ import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
 import apiFetch from '@wordpress/api-fetch';
 import { useCallback, useMemo, useState } from 'react';
 
+const calculateReappearanceTime = minutes => {
+	if ( minutes === -1 ) {
```

Comment:
May be add a comment here saying -1 means never show the notice again.

Reply:
I added the comment on the `dismissNotice` function, because that's what you call, this one is only an internal function used by `dismissNotice`, but I can comment here too!